core/storage/suggestion/gae_models.py

```diff
@@ -297,6 +297,27 @@ def query_suggestions(cls, query_fields_and_values):
 
         return query.fetch(feconf.DEFAULT_QUERY_LIMIT)
 
+    @classmethod
+    def get_translation_suggestions_in_review_by_exp_id(cls, exp_id):
+        """Returns translation suggestions which are in review with target_id
+        == exp_id.
+
+        Args:
+            exp_id: str. Exploration ID matching the target ID of the
+                translation suggestions.
+
+        Returns:
+            list(SuggestionModel). A list of translation suggestions in review
```

Comment:
Suggest also mentioning that the number of returned results is limited by feconf.DEFAULT_QUERY_LIMIT (this is a potentially important behavior that may affect upstream dependencies).

Reply:
Done!